generators/generator-base.js

```diff
@@ -41,7 +41,7 @@ util.inherits(Generator, yeoman.Base);
  */
 Generator.prototype.addElementToMenu = function (routerName, glyphiconName, enableTranslation) {
     try {
-        var fullPath = CLIENT_MAIN_SRC_DIR + 'app/layouts/navbar/navbar.html';
+        var fullPath = CLIENT_MAIN_SRC_DIR + 'app/layouts/navbar/navbar.component.html';
```

Comment:
just renaming wouldnt work as the code in both version would be slightly different as well, so we would have to do conditions here anyway